src/Agent/NewRelic/Agent/Core/Logging/AuditLog.cs

```diff
@@ -13,21 +13,23 @@ public static class AuditLog
         private static Lazy<ILogger> _lazyAuditLogger = new Lazy<ILogger>(() =>
             Serilog.Log.Logger.ForContext(LogLevelExtensions.AuditLevel, LogLevelExtensions.AuditLevel));
 
+        public static bool IsAuditLogEnabled { get; private set; }
+
         /// <summary>
         /// Logs <paramref name="message"/> at the AUDIT level. This log level should be used only as dictated by the security team to satisfy auditing requirements.
         /// </summary>
         public static void Log(string message)
         {
+            if (IsAuditLogEnabled)
                 // use Fatal log level to ensure audit log messages never get filtered due to level restrictions
                 _lazyAuditLogger.Value.Fatal(message);
         }
 
         internal static LoggerConfiguration IncludeOnlyAuditLog(this LoggerConfiguration loggerConfiguration)
         {
-            return loggerConfiguration.Filter.ByIncludingOnly($"{LogLevelExtensions.AuditLevel} is not null");
+            IsAuditLogEnabled = true; // set a flag so we can short-circuit when audit log is not enabled
```

Comment:
Should we add a test that ensures `IsAuditLogEnabled` is true when this method is called?